kibble/render/render.go

```diff
@@ -69,8 +69,6 @@ func Watch(sourcePath string, buildPath string, cfg *models.Config, port int32,
 
 // Render - render the files
 func Render(sourcePath string, buildPath string, cfg *models.Config) int {
-	
-	log.Infof("UseTranslationsApi: %t", cfg.UseTranslationsApi)
```

Comment:
I think it's worth keeping this for a while longer.

Reply:
It doesn't do anything though... it will always be false because we aren't using that in any kibble.json files.  And the logic is wrong anyway... kibble.json shouldn't be making decisions about using the translations api... the feature toggle decides that.

Reply:
Yes, you're right, but I was meaning the intention of knowing if a site is being built with translations coming from the API/local files - I still want to know that while this is bedding in. So if we can get the feature toggle being logged...

Reply:
@brynyShift72 I still want this.

Reply:
@brynyShift72 he still wants this!

Reply:
@brynyShift72 nvm I see it :/